project/project.json5

```diff
@@ -53,6 +53,29 @@
                 ],
             },
         },
+        {
+          dataset_id: 'eia_861_annual_energy_use_state_sector',
+          dataset_type: 'historical',
+          version: '1.0.0',
+          required_dimensions: {
+          single_dimensional: {
+          model_year: {
+            base: [
+              "2010","2011","2012","2013","2014","2015","2016","2017","2018","2019","2020","2021","2022"
```

Comment:
This raises a good question: Which model years do we want to have access to in the project? 

That probably depends on what approach we want to take in calculating residuals which we haven't figured out yet. Maybe for now we should expand the project model years back to 2012; however, if we do that we'll have to adjust other datasets' mapping files and required dimensions. 

Another approach we could take is to only map one or a few years of EIA data into the decarb project, but also establish an EIA-dataset only project for calculating, e.g., growth rates we might want to use in the main project ...

Any thoughts @ashreeta @lixiliu @daniel-thom?

Reply:
I think only mapping one year of EIA data makes more sense - the year could be 2024 created by adjusting the final year EIA data with a growth based on the same trajectory as the other historical years. Alternatively we could just use 2022 and only shift back project years to 2022. I don't know if we have sensible mappings for all datasets for historical years back to 2012 but 2022 might be more realistic.